lib/portlayer/storage/data_source.go

```diff
@@ -70,7 +77,7 @@ func (m *MountDataSource) Export(op trace.Operation, spec *archive.FilterSpec, d
 
 	// NOTE: this isn't actually diffing - it's just creating a tar. @jzt to explain why
```

Comment:
It relies on docker's `ChangesDirs` func in the archive package. It takes two directories, new and old, and produces a list of the differences. If you supply `""` as the old directory, it will basically diff against nothing, so you end up with everything.